docs/src/main/asciidoc/_developer-guide/Remote_Jakarta_Enterprise_Beans_invocations_via_JNDI_-_Jakarta_Enterprise_Beans_client_API_or_wildfly-naming-client_project.adoc

```diff
@@ -1,59 +1,33 @@
-[[Remote_Jakarta_Enterprise_Beans_invocations_via_JNDI_-_Jakarta_Enterprise_Beans_client_API_or_remote-naming_project]]
-= Remote Jakarta Enterprise Beans invocations via JNDI - Jakarta Enterprise Beans client API or remote-naming project
+[[Remote_Jakarta_Enterprise_Beans_invocations_via_JNDI_-_Jakarta_Enterprise_Beans_client_API_or_wildfly-naming-client_project]]
+= Remote Jakarta Enterprise Beans invocations via JNDI - Jakarta Enterprise Beans client API or wildfly-naming-client project
 
 [[purpose]]
 == Purpose
 
-WildFly provides EJB client API project as well as remote-naming project
+WildFly provides EJB client API project as well as wildfly-naming project(https://github.com/wildfly/wildfly-naming-client)
```

Comment:
small issue but it should be: wildfly-naming-client project, there is also a number of such typos below; I believe those should be changed

Reply:
Thanks for the update @tadamski, I'll change it accordingly.

Reply:
Done @tadamski